plugins/modules/aci_hsrp_interface_policy.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_hsrp_interface_policy
+short_description: Manage HSRP interface policies (hsrp:IfPol)
+description:
+- Manage HSRP interface policies on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of the Tenant the hsrp interface policy should belong to.
+    type: str
+    aliases: [ tenant_name ]
+    required: true
```

Comment:
don't think this is required for class query

Reply:
I removed it.